superset/views/sql_lab.py

```diff
@@ -242,7 +243,7 @@ def delete_query(  # pylint: disable=no-self-use
                 .filter(
                     and_(
                         Query.client_id != client_id,
-                        Query.user_id == g.user.get_id(),
+                        Query.user_id == get_user_id(),
```

Comment:
Mixed types, though likely benign from the database's perspective—at least for MySQL.